src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -247,20 +247,7 @@ def detect_dotnet_version_tocreate(detected_ver):
 def detect_node_version_tocreate(detected_ver):
     if detected_ver in NODE_VERSIONS:
         return detected_ver
-    # get major version & get the closest version from supported list
-    major_ver = int(detected_ver.split('.')[0])
```

Comment:
any reason we are removing all this? we should for NODE with the new versions we support if its >10 or something.